tests/integration/targets/vmware_esxi_connection/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+test_esxi_hostname: "{{ tiny_prefix }}_esxi_connection"
+test_resource_pool: "{{ tiny_prefix }}_esxi_connection_rp"
+test_cluster: Eco-NestedCluster
```

Comment:
Use Eco-Cluster, please. Eco-NestedCluster was created for another purpose and I'm not sure it will remain